webrtc.html

```diff
@@ -1290,9 +1291,19 @@
                       <li>
                         <p>If the direction of the media description is
                         <code>sendrecv</code> or <code>sendonly</code>, and
-                        <code>transceiver.receiver.track</code> has not yet
-                        been fired in a <code title="event-track"><a>track
-                        </a></code> event, <a>process the remote track</a> for
+                        <var>transceiver</var>'s
+                        [[<a href="#direction-slot">Direction</a>]] slot is
```

Comment:
This condition seems backwards. Instead of `[[Direction]] != "sendrecv" && [[Direction]] != "sendonly"`, shouldn't it be `[[CurrentDirection]] != "sendrecv" && [[CurrentDirection]] != "recvonly"`? In other words, "last time we negotiated, I didn't offer to receive a track from you"?